jwala-integration-test/src/test/postman/jwala-collection.postman_collection.json

```diff
@@ -6550,7 +6688,7 @@
 					],
 					"request": {
 						"url": {
-							"raw": "{{url}}/v1.0/applications/fileExists?filePath=D:\\stp\\app\\webapps\\{{HealthCheckWar}}&hostName={{host}}",
+							"raw": "{{url}}/v1.0/applications/fileExists?filePath=D:\\ctp\\app\\webapps\\{{helloWorldWar}}&hostName={{host}}",
```

Comment:
This path needs to be derived somehow. If the goal is to make the tests generic then the webapps path can't be hard coded with any drive info - how does this pass when running on Linux?